torch/_inductor/mkldnn.py

```diff
@@ -93,11 +93,10 @@ def check_binary_op_kwargs_is_default(node):
     return True
 
 
-class ConvUnary2d(nn.Conv2d):
+class PackedConv2d(nn.Conv2d):
```

Comment:
A side note: we leave weight repacking before AOT autograd while move fusion after it. In the future, we may consider to move weight prepacking after AOT autograd too and do it together with constant folding passes there.